### DISCUSSION TITLE
怎么在Github Action打包本地文件？

### DISCUSSION THREAD

**Discussion by dyxang** [tw93/Pake#528] [Q&A] [answered]
> url 是您需要打包的网页链接 🔗 或本地 HTML 文件的路径，此参数为必填。

1. 我在仓库中新建了一个`fmg`文件夹，本地html文件放里面.

2. 然后按[在线编译（普通用户使用）](https://github.com/tw93/Pake/wiki/%E5%9C%A8%E7%BA%BF%E7%BC%96%E8%AF%91%EF%BC%88%E6%99%AE%E9%80%9A%E7%94%A8%E6%88%B7%E4%BD%BF%E7%94%A8%EF%BC%89)的2方法URL填`./fmg/index.html`结果报错 
```bash
error: command-argument value '.\fmg\index.html' is invalid for argument 'url'. Your url "https://.\fmg\index.html" is invalid
```

另外这种时候怎么使用`--copy-iter-file`?

**Comment by Tlntin** [comment, marked as answer]
action貌似不支持本地文件夹打包，你实在要打包的话，看看文档，需要改一些配置。

**Reply by dyxang** [reply]
好的，另外bin/README_CN.md 里的 --copy-iter-file 是不是应该改成 --iter-copy-file？

**Comment by Tlntin** [comment]
> 好的，另外bin/README_CN.md 里的 --copy-iter-file 是不是应该改成 --iter-copy-file？

好的，谢谢提醒，已经修改了。
